### PR TITLE
Activity setup

### PR DESCRIPTION
**Pull request by EmilChri**
Activity module setup  has been completed and tested.
It is set up such that that pages for quiz creation can be made.
Link explaining what the different files do: https://docs.google.com/document/d/1kWcjeOjVhe4WW28KSwYdGomw4ry5gKCBfKrtG6CuPRg/edit

### CODE REVIEW

**Review by CarlHejlesen — COMMENTED** [review]
Jeg mener det ser meget fornuftigt ud.
Følger moodles snake case.
Går ud fra det er testest på alle i gruppens computere.
Følger deres dokumentation inde for db og hvor plugin skal være.
Fin indsats :D

**Comment by noobstrikker** [comment]
+points for at update .gitingore efter pratisk brug

File: .idea/workspace.xml
Comment (DeeKahy) [inline .idea/workspace.xml:1]: i skal tilføje den her til git ignore.

**Review by DeeKahy — CHANGES_REQUESTED** [review]
Is there a good reason to add the workspace.xml? I normally just add the entire .idea folder to the gitignore and untrack it if its tracked.

**Review by ChristofferGamel — COMMENTED** [review]
Ved ikke, hvad .idea debatten går ud på, men ellers LGTM

File: .idea/workspace.xml
Comment (adomasciplys) [reply .idea/workspace.xml:1]: Has been fixed in 1644dfd53cd2728ab68567e307287322211a823e. Ready for merge after :)

**Review by DeeKahy — CHANGES_REQUESTED** [review]
This didnt actually delete the file. you have to untrack it.

**Review by DeeKahy — APPROVED** [review]
Very nice <3

**Pull request merged**